packages/react/src/components/Video/Video.tsx

```diff
@@ -0,0 +1,95 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import { createShorthandFactory, UIComponent, UIComponentProps, commonPropTypes } from '../../lib'
+import { videoBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+
+import { ReactProps } from '../../types'
+
+import VideoGif from './VideoGif'
+
+export interface VideoProps extends UIComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default videoBehavior
+   * */
+  accessibility?: Accessibility
+
+  /** Video source URL. */
+  src?: string
+
+  /** Image source URL for when video isn't playing */
+  poster?: string
+
+  /** Whether the video should be allowed to play audio */
+  muted?: boolean
+
+  /** Whether the video should start playing when rendered. Autoplay videos must be muted or they will not play immediately in certain browers like Chrome */
+  autoPlay?: boolean
+
+  /** Whether to display the native video controls */
+  controls?: boolean
+
+  /** Whether the video should automatically restart after it ends */
+  loop?: boolean
+}
+
+/**
+ * An video is a graphicical and audio representation of something.
+ * @accessibility
```

Comment:
I suggest to drop accessibility part for initial implementation at all and introduce it later with sync with our accessibility team. What do you think?

Reply:
Sounds good for Video. For Video Gif, I'd like to keep in the key actions so that space/enter on GIFs still play/pause them. But if there's some issue there too we can take that up after the accessibility sync.